src/cameras/PerspectiveCamera.js

```diff
@@ -135,6 +151,18 @@ THREE.PerspectiveCamera.prototype.copy = function ( source ) {
 	this.near = source.near;
 	this.far = source.far;
 
+	this.zoom = source.zoom;
+	this.focalLength = source.focalLength;
+
+	this.skewSlopeX = source.skewSlopeX;
+	this.skewSlopeY = source.skewSlopeY;
+
+	if ( source.view !== null && source.view !== undefined ) {
```

Comment:
For the record the non-triple not-equals matches both null and undefined but if you want to check only null it's a moot point.
I'm unclear why not using undefined isn't prefered?  Best guess null is not an object and undefined is so it takes more resources unessecarily? 
!= undefined covers null too.